examples/cargo_ex_mutex.c

```diff
@@ -117,7 +117,6 @@ int main(int argc, char **argv)
         return -1;
     }
 
-fail:
```

Comment:
The correct solution would be to replace `return -1` above with `goto fail;` ... oh and add `int ret = 0;`at top